DEVELOPERS.md

```diff
@@ -4,10 +4,13 @@
 * [Installing Dependencies](#Installing-Dependencies)
 * [Building Supabase](#Building-Supabase)
 * [Start a Development Server](#Start-a-Development-Server)
-* [Build in Public](#Build-in-Public)
+* [What's Next?](#What's-Next)
```

Comment:
Can you change this to something else, the link doesn't work and needs to be changed.

Reply:
Yup, I have changed and it works